### DISCUSSION TITLE
Correct way to install Hyas and use starter themes

### DISCUSSION THREAD

**Discussion by pensivedog** [gethyas/hyas#398] [General]
I just used ```npm create hyas@latest``` and selected the default-starter. After running ```npm  install```, three high severity vulnerabilities are found which cannot be fixed with ```npm audit fix```.

The setup instructions on gethyas.com still say to clone the repo, rather than use ```npm create hyas@latest```, as described in the repo readme.

Can you please clarify how to proceed if we want to setup and deploy a new Hyas-based site? Are sites created using ```npm create hyas@latest``` and choosing a starter theme ready for use? I know from [another discussion thread](https://github.com/orgs/gethyas/discussions/372) that the Doks starter theme isn't actually ready and will fail in production.

Are the other starter themes offered during setup ready for use in production?

**Comment by h-enk** [comment]
Thanks!

> I just used npm create hyas@latest and selected the default-starter. After running npm  install, three high severity vulnerabilities are found which cannot be fixed with npm audit fix.

I missed that: I've now released  [gethyas v2.0.5](https://github.com/gethyas/hyas/releases/tag/v2.0.5). Now running `npm update gethyas` will show zero vulnerabilities found. I will update [create-hyas](https://github.com/gethyas/create-hyas) soonish.

> The setup instructions on gethyas.com still say to clone the repo, rather than use npm create hyas@latest, as described in the repo readme.

The recommended way is to use `npm create hyas@latest`. Only the Doks theme (preview release v1.0.0-alpha.1) is not production ready yet (but will be soon). Doks v0.5 is production ready (and available via `git clone`).

The Hyas and Doks sites will be updated soon (when Doks v1.0.0-beta.1 is available) and accordingly

**Reply by pensivedog** [reply]
Just used `npm create hyas@latest` and found the installed `@hyas` packages are not the latest releases listed in the Hyas readme file. Sorry, I'm new to npm... shall I just update the version numbers in `package.json`, add the missing packages, and then run `npm update`? Thanks!

**Reply by h-enk** [reply]
When you run `npm outdated`, you see the packages that need updating.

Then, you could manually edit `package.json`, run `npm run clean:install`, and next `npm i` (that's what I prefer).

Or, just run `npm update`.

Note, in about 4 hours from now I will be releasing a new version of [create-hyas](https://github.com/gethyas/create-hyas)

**Reply by h-enk** [reply]
New [create-hyas](https://github.com/gethyas/create-hyas) v1.0.3 is available now!
